site/content/docs/main/csi-snapshot-data-movement.md

```diff
@@ -0,0 +1,390 @@
+- For public cloud users, this feature helps users to fulfil the multiple cloud strategy. It allows users to back up volume snapshots from one cloud provider and preserve or restore the data to another cloud provider. Then users will be free to flow their business data across cloud providers based on Velero backup and restore.  
+
+Besides, Velero [File System Backup][3] which could also back up the volume data to a pre-defined backup storage. CSI Snapshot Data Movement works together with [File System Backup][3] to satisefy different requirements for the above scenarios. And whenever available, CSI Snapshot Data Movement should be used in preference since the [File System Backup][3] reads data from the live PV, in which way the data is not captured at the same point in time, so is less consistent.  
+Moreover, CSI Snapshot Data Movement brings more possible ways of data access, i.e., accessing the data from the block level, either fully or incrementally.  
+On the other hand, there are quite some cases that CSI snapshot is not available (i.e., you need a volume snapshot plugin for your storage platform, or you're using EFS, NFS, emptyDir, local, or any other volume type that doesn't have a native snapshot), then [File System Backup][3] will be the only option.  
+
+CSI Snapshot Data Movement supports both built-in data mover and customized data movers. For the details of how Velero works with customized data movers, check the [Volume Snapshot Data Movement design][1]. Velero provides a built-in data mover which uses Velero built-in uploaders (at present the available uploader is Kopia uploader) to read the snapshot data and write to the Unified Repository (by default implemented by Kopia repository).    
+
+The way for Velero built-in data mover to access the snapshot data is based on the hostpath access by Velero node-agent, so the node-agent pods need to run as root user and even under privileged mode in some environments, as same as [File System Backup][3].  
+
+## Setup CSI Snapshot Data Movement
+
+## Prerequisites
+
+ 1. The source cluster is Kubernetes version 1.20 or greater.
+ 2. The source cluster is running a CSI driver capable of support volume snapshots at the [v1 API level][4].
+ 3. CSI Snapshot Data Movement requires the Kubernetes [MountPropagation feature][5].
+
+
+### Install Velero Node Agent
+
+Velero Node Agent is a Kubernetes daemonset that hosts Velero data movement modules, i.e., data mover controller, uploader & repository. 
+If you are using Velero built-in data mover, Node Agent must be installed. To install Node Agent, use the `--use-node-agent` flag. 
+
+```
+velero install --use-node-agent
+```
+
+### Configure Node Agent DaemonSet spec
```

Comment:
`Configure Node Agent DaemonSet spec` - This whole section is similar to what needs to be done for file system backup, right ? maybe we just link it here ?

Reply:
Yes, they are almost the same for fs backup and data mover right now. However, things may be different in future along with the evolvement. For example, the block mode data mover will require a different host path.

Therefore, I would prefer keeping them independent.